classes/orcid/OrcidManager.php

```diff
@@ -363,4 +334,29 @@ private static function writeLog(string $message, string $level): void
         $logFilePath = Config::getVar('files', 'files_dir') . '/orcid.log';
         error_log("{$fineStamp} {$level} {$message}\n", 3, $logFilePath);
     }
+
+    /**
+     * Gets the ORCID API endpoint to revoke an access token
+     */
+    public static function getTokenRevocationUrl(?Context $context = null): string
```

Comment:
`$context` isn't used so it can be removed from the function arguments.

Reply:
updated